src/PrestaShopBundle/Form/Admin/Type/CurrencyChoiceType.php

```diff
@@ -27,35 +27,66 @@
 
 namespace PrestaShopBundle\Form\Admin\Type;
 
+use PrestaShop\PrestaShop\Core\Currency\CurrencyDataProviderInterface;
 use PrestaShop\PrestaShop\Core\Form\ChoiceProvider\CurrencyByIdChoiceProvider;
+use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\Options;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class CurrencyChoiceType extends ChoiceType
+class CurrencyChoiceType extends AbstractType
 {
     /**
      * @var CurrencyByIdChoiceProvider
      */
     private $currencyByIdChoiceProvider;
+    /**
```

Comment:
```suggestion

    /**
```

Reply:
(not mandatory though, should in theory be detected by cs fixer, so if there are no other things to fix, just ignore this :arrow_up: )